Topic 0/src/point2/Demo.java

```diff
@@ -0,0 +1,14 @@
+package point2;
+
+public class Demo {
+
+	public static void main(String[] args) {
+		
+		Builder builder = new LivingRoom();
```

Comment:
[High] In the way you have define it, the Architect will never get the complete Build (because you loose the built room every time that you assign a new roomBuilder).

Your "product" object should be a House, not just a Room as you have modelate it. That's why you can't get the complete product when you build the object.

A House could be composed by several rooms.